engine/glfw/lib/ios/app/BaseView.m

```diff
@@ -477,6 +488,7 @@ - (void)deleteBackward
         [markedText setString:@""];
         _glfwSetMarkedText("");
     } else {
+        _glfwInputKey( GLFW_KEY_BACKSPACE, GLFW_RELEASE );
```

Comment:
hack out repeat system to make sure we remove a bit faster then currently possible
there is no long press event for virtual keyboard